tensorflow_addons/image/median_filter_2D_test.py

```diff
@@ -0,0 +1,95 @@
+import tensorflow as tf
+import median_filter_2D as md
+
+
+
+class Median2DTest(tf.test.TestCase):
+
+
+  def _validateMedian_2d(self, inputs, expected_values, filter_shape = (3, 3)):
```

Comment:
Can you run the tests against eager and graph mode? There is a decorator that is used in the other test files

Reply:
can you elaborate on the decorator? I did not get it.

Reply:
https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/maxout_test.py#L28 For example

Reply:
@seanpmorgan Can you guide me to create placeholder with shape = None using tensorflow 2.0. placeholder i think is removed from 2.0. How to create a None shape tensor.?

Reply:
Error while using the decorator in TF 2.0 as the new tensorflow addon is supported with TF 2.0. 

Issue updated at #141 
Please have a look.

Reply:
Got a workaround. Executed test for graph and eager moth differnently. That fixed it. Solved.